examples/sandbox/index.ts

```diff
@@ -28,10 +30,14 @@ import * as dashTxJson from './json/dashTx.json'
 import * as dogeTxJson from './json/dogeTx.json'
 import * as ltcTxJson from './json/ltcTx.json'
 
+import Portis from "@portis/web3";
+
 const keyring = new Keyring()
 
+const portis = new Portis('ff763d3d-9e34-45a1-81d1-caa39b9c64f9', 'mainnet');
```

Comment:
This should be part of the adapter similar to how Trezor is implemented.

Reply:
Its created here to pass it thru to HD wallet

Reply:
@elmutt The point of the adapter is to encapsulate all this logic. see @keepkeyjon comments above

```
      const portisAdapter = PortisAdapter.useKeyring(keyring, config)
```

Reply:
This is the sandbox

Reply:
Yep.  thats a different line.  im doing something similar with:

```
const portisAdapter = PortisAdapter.useKeyring(keyring, config)
```

Reply:
ok i . see what u mean.  was confused what line u meant